website/content/docs/drivers/podman.mdx

```diff
@@ -49,39 +43,31 @@ The `podman` driver implements the following [capabilities](/docs/internals/plug
 ## Known Limitations
 
 The Podman task driver is under active development. It currently does not support [stderr logging][stderr-logging] and [devices][devices].
-Podman recently released [Podman v2](https://podman.io/blogs/2020/06/29/podman-v2-announce.html). The task driver currently supports v1
-and will be working on supporting v2 in upcoming releases.
 
 ## Task Configuration
 
 Due to Podman's similarity to Docker, the example job created by [`nomad init -short`][nomad-init] is easily adapted to use Podman instead:
 
 ```hcl
-job "example" {
+``job "redis" {
```

Comment:
```suggestion
job "redis" {
```

Reply:
Err, I'm not sure what's happening with that suggestion... but remove the extra code fence chars is what I mean.